nflow-tests/src/test/java/io/nflow/tests/StateVariablesTest.java

```diff
@@ -105,6 +106,7 @@ public void updateStateVariable() {
 
   @Test
   @Order(4)
+  @DisabledIfSystemProperty(named = "profiles", matches = "nflow.db.[postgresql|sqlserver|oracle]")
```

Comment:
Would it be better to use `assumeThat(lengthOfStateVarField, lessThan(10500))` that way the list of databases is not hardcoded but instead the constants of the test is hardcoded?

Reply:
Yes, updated.